packages/vscode-dependencies-validation/src/util.ts

```diff
@@ -24,28 +24,8 @@ export async function findAndFixDepsIssues(
 ): Promise<void> {
   const { fsPath } = packageJsonUri;
   const depIssues = await findDependencyIssues(fsPath);
-  const problems = depIssuesToProblemStrings(depIssues);
-  if (isEmpty(problems)) return;
-
+  if (isEmpty(depIssues)) {
```

Comment:
jetbrains does not seem to support smart breakpoints within the same line
so i cannot debug it without the block